apps/backend/src/datasources/postgres/FapDataSource.ts

```diff
@@ -779,6 +797,80 @@ export default class PostgresFapDataSource implements FapDataSource {
     return fapUpdated;
   }
 
+  async assignMemberToFapProposals(
```

Comment:
This looks very similar to the existing `assignMemberToFapProposal`. I wonder if it might be better to change that method to something like `assignMembersToFapProposals(assignments: {proposalPk, fapId, memberId}[])` and tweak the places that call it instead of near duplicating it?

Reply:
Good point. Would it be an idea to open a separate issue to do that? Having a look, this change would make the PR a fair bit bigger and harder to review I reckon.

Reply:
Sure, that sounds reasonable